src/test/java/jewellery/inventory/integration/AuthIntegrationTest.java

```diff
@@ -35,6 +33,7 @@ class AuthIntegrationTest extends AuthenticatedIntegrationTestBase {
   @BeforeEach
   void setUp() {
     this.setup();
+    userRepository.deleteAll();
```

Comment:
We have this method in AuthenticatedIntegrationTestBase::setup. Why do we need to delete the users again here?